src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_AMOUNT = 5;
+    private static final double RANDOM_PROPERTIES = 25;
+    private static final double DEFAULT_CIRCLE_RADIUS = 10;

```

Comment:
You won't get a random value, because we have it fixed and goes to all figures.
The value with the sides of the figure should be obtained randomly through Random